easybuild/easyconfigs/u/UCX/UCX-1.5.0rc1-GCCcore-8.2.0-hpcx.eb

```diff
@@ -0,0 +1,44 @@
+# Note:
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.5.0rc1'
+versionsuffix = '-hpcx'
+
+homepage = 'http://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openucx/ucx/releases/download/v1.5.0-rc1']
+sources = ['ucx-1.5.0.tar.gz']
```

Comment:
Hmmmm, important note here, since you're pulling down 1.5.0-rc1 the source should file not be called ucx-1.5.0.tar.gz.
You need to do it like this or there will be checksum conflicts for people who already have the real 1.5.0 downloaded.
```
sources = [
    {'filename': 'ucx-1.5.0-rc1.tar.gz', 'download_filename', 'ucx-1.5.0.tar.gz'}
]
```

Reply:
Done.